__builtin_ia32_vcvtph2ps256 has been removed in newer Clang versions.
Use __builtin_convertvector instead when compiling with Clang >= 15.

Backported from upstream Skia commit 9ab05b08ae3e83c80ac69c69008ed1048e9a0923
(which rolled skcms 238fd0489c04).
---
 modules/skcms/src/Transform_inl.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/modules/skcms/src/Transform_inl.h b/modules/skcms/src/Transform_inl.h
index bc10e7d47d..aad05750a6 100644
--- a/modules/skcms/src/Transform_inl.h
+++ b/modules/skcms/src/Transform_inl.h
@@ -166,8 +166,13 @@ SI F F_from_Half(U16 half) {
     return (F)_mm512_cvtph_ps((__m256i)half);
 #elif defined(USING_AVX_F16C)
+#if defined(__clang__) && __clang_major__ >= 15
+    typedef _Float16 __attribute__((vector_size(16))) F16;
+    return __builtin_convertvector((F16)half, F);
+#else
     typedef int16_t __attribute__((vector_size(16))) I16;
     return __builtin_ia32_vcvtph2ps256((I16)half);
+#endif // defined(__clang))
 #else
     U32 wide = cast<U32>(half);
     // A half is 1-5-10 sign-exponent-mantissa, with 15 exponent bias.
